### PR TITLE
Added a new Question

### PR DESCRIPTION
**Pull request by abhishekiiitr**
Type of variable in Python

Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/python-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+      {
+    Question: 
+      "In Python,if we have a='abhi' , then what is the result here : print(type(a))?",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:4]:
```suggestion
      "In Python, if organization='freeCodeCamp', then what is the result here: print(type(organization))?",
```

File: src/data/python-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,15 @@
...
+    Answer: "<class 'str'>",
+    Distractor1: "<class 'char'>",
+    Distractor2: "<class 'var'>",
+    Distractor3: "error",
+    Explanation: 
+      "type() method returns class type of the argument(object) passed as parameter in Python",
```
Comment (jdwilkin4) [inline src/data/python-quiz.ts:10]:
```suggestion
      "Python's type() method returns the class type for the object.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by abhishekiiitr — COMMENTED** [review]
I have made the required changes, kindly merge the PR !

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**